register.js

```diff
@@ -0,0 +1,54 @@
+
+
+document.getElementById('submitButton').addEventListener('click', (e) => {
+    //Prevents default form action from happening
+    e.preventDefault();
+
+    //Get Previous Items as array
+    let previousClients = getLocalStorage()
+    
+    console.log(previousClients, "Previous Clients")
+
+    //Get Input Values
+    let nameValue = document.querySelector('input[type=text]').value;
+    let emailValue = document.querySelector('input[type=mail]').value;
+    let phoneValue = document.querySelector('input[type=tel]').value;
+    validateForm(nameValue,emailValue,phoneValue)
+
+
+    let clientObject = {
+        'name': nameValue,
+        'email': emailValue,
+        'phoneNumber': phoneValue
+    }     
+
+    //add to previous storage array
+    previousClients.push(clientObject)
+
+
+    //store in local storage
+    localStorage.setItem('employees', JSON.stringify(previousClients))
```

Comment:
Why did you use stringify to insert the object in the local storage?

Reply:
since localStorage can only store strings, so we convert JSON object to string.

Reply:
Cool